ods-devenv/scripts/deploy.sh

```diff
@@ -1489,6 +1489,8 @@ function create_configuration() {
     sed -i "s|PROV_APP_CONFLUENCE_ADAPTER_ENABLED=.*$|PROV_APP_CONFLUENCE_ADAPTER_ENABLED=false|" ods-core.env
     sed -i "s|PROV_APP_AUTH_BASIC_AUTH_ENABLED=.*$|PROV_APP_AUTH_BASIC_AUTH_ENABLED=true|" ods-core.env
     sed -i "s|PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=.*$|PROV_APP_PROVISION_CLEANUP_INCOMPLETE_PROJECTS_ENABLED=true|" ods-core.env
+    sed -i "s|PROV_APP_OPENSHIFT_SERVICE_ENABLED=.*$|PROV_APP_OPENSHIFT_SERVICE_ENABLED=true|" ods-core.env
```

Comment:
@stitakis  - can you please set to false!

@michaelsauter

Reply:
Same opinion as @michaelsauter:  
> The AMI build should reflect reality as much as possible. If we disable functionality that is used in real installations, we have not tested it ...

Reply:
now that I understand it, ok